pkg/apis/mattermost/v1alpha1/clusterinstallation_utils.go

```diff
@@ -135,6 +135,11 @@ func (mattermost *ClusterInstallation) GenerateService() *corev1.Service {
 
 // GenerateIngress returns the ingress for Mattermost
 func (mattermost *ClusterInstallation) GenerateIngress() *v1beta1.Ingress {
+	ingressAnnotations := map[string]string{}
```

Comment:
mattermost.Spec.IngressAnnotations is already a map[string]string, so we should just set `Annotations:  mattermost.Spec.IngressAnnotations` below.

Reply:
done